pkg/server/server.go

```diff
@@ -801,7 +801,7 @@ func validateAdmin(imageSpace string, w http.ResponseWriter, r *http.Request) bo
 	if err != nil {
 		message := fmt.Sprintf("Unable to get permission token %s", err)
 		kiln.LogError.Printf(message)
-		writeErrorResponse(http.StatusUnauthorized, message, w)
```

Comment:
I agree. And upon looking at it more there is still an area above on line 793 that would return a 401. I'll look into the implementation of `authsdk.NewJWTTokenFromRequest` and see if we can parse the error to give us more information.